.gitlab-ci.yml

```diff
@@ -82,6 +83,11 @@ nfs-itest:
   stage: nfs-itest
   script: gradle :submission:submission-webapp:itest -PenableFire=false --rerun-tasks
 
+nfs-secret-key-test:
+  needs: [ "build-test" ]
+  stage: nfs-itest
+  script: gradle :submission:submission-webapp:itest --tests SubmissionReleaseTest -PincludeSecretKey=true -PnfsReleaseMode=MOVE --rerun-tasks
```

Comment:
I think a completly whole new test phase only for a property is too much, I think we can create single couple of test that test this.

Reply:
Do you mean, leave the test but not run it in the CI? My idea was to run this stage only temporarily, while we leave the secretKey/move behaviour as default so the feature flags will be no longer needed